sympy/printing/latex.py

```diff
@@ -289,6 +289,10 @@ def _do_exponent(self, expr, exp):
         else:
             return expr
 
+    def _print_Basic(self, expr):
+        l = [self._print(o) for o in expr.args]
+        return expr.__class__.__name__ + "(%s)" % ", ".join(l)
```

Comment:
It should latexify the class name (I think the function you want is `self._deal_with_super_sub`).